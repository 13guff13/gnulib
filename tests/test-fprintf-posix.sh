#!/bin/sh

tmpfiles=""
trap 'rm -fr $tmpfiles' 1 2 3 15

tmpfiles="$tmpfiles t-fprintf-posix.tmp"
./test-fprintf-posix${EXEEXT} > t-fprintf-posix.tmp || exit 1

: ${DIFF=diff}
${DIFF} "${srcdir}/test-fprintf-posix.out" t-fprintf-posix.tmp
result=$?

rm -fr $tmpfiles

exit $result
