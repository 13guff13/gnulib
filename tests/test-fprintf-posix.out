12345671 33
12345672 33
12345673 33
0x0p+0 33
-0x0p+0 33
inf 33
-inf 33
nan 33
       inf 33
       nan 33
12.750000 33
1234567.000000 33
-0.031250 33
0.000000 33
-0.000000 33
nan 33
00001234.000000 33
1234 33
12.750000 33
1234567.000000 33
-0.031250 33
0.000000 33
-0.000000 33
nan 33
00001234.000000 33
1234 33
12.750000 33
1234567.000000 33
-0.031250 33
0.000000 33
-0.000000 33
NAN 33
00001234.000000 33
1234 33
12.750000 33
1234567.000000 33
-0.031250 33
0.000000 33
-0.000000 33
NAN 33
00001234.000000 33
1234 33
55 33
